apps/src/templates/page.html.ejs

```diff
@@ -78,6 +78,7 @@
 </div>
 
 <div id="visualizationResizeBar">
+  <i id="visualizationResizeIcon" class="fa fa-ellipsis-v"></i>
```

Comment:
fa is "font awesome". here is what they have to say on the topic: "You can place Font Awesome icons just about anywhere using the CSS Prefix fa and the icon's name. Font Awesome is designed to be used with inline elements (we like the `<i>` tag for brevity, but using a `<span>` is more semantically correct)." http://fortawesome.github.io/Font-Awesome/examples/